trpcultivate_phenotypes/src/Plugin/TripalImporter/TripalCultivatePhenotypesTraitsImporter.php

```diff
@@ -150,16 +196,16 @@ public function configureValidators(array $form_values, string $file_mime_type)
 
     $validators = [];
 
-    // Setup the plugin manager
+    // Setup the plugin manager.
     $manager = \Drupal::service('plugin.manager.trpcultivate_validator');
```

Comment:
Addressed in commit [178f162](https://github.com/TripalCultivate/TripalCultivate-Phenotypes/pull/122/commits/178f162fc69f0ff2fdeedf284c8b758eb30e6a83) and [a556752](https://github.com/TripalCultivate/TripalCultivate-Phenotypes/pull/122/commits/a556752611426cc80ba0c2f1f597fb498fde2efa). 

However there are still 2 service calls in `describeUploadFileFormat()`:
```
$file_link = \Drupal::service('trpcultivate_phenotypes.template_generator')
      ->generateFile($importer_id, $column_headers);
```
and
```
return \Drupal::service('renderer')->renderPlain($build);
```

I wonder if these should be addressed in a separate PR since as far as I can tell there are currently no automated tests for this method, so I'm nervous to attempt dependency injection for these without also creating a test to ensure it was done properly. Also, is the renderer service the same as the one being used in PR #114?

Reply:
After conversation offline with Carolyn, we decided that I would quickly fix the last two instances. This has been done in commit https://github.com/TripalCultivate/TripalCultivate-Phenotypes/pull/122/commits/5c073e9b308466ebeb9fb9f03dbccf354e678dae.